{"c775de6b3c3abab8ee894156409015f5":{"statistic":{"failed":0,"broken":0,"skipped":0,"passed":1,"unknown":0,"total":1},"items":[{"uid":"50c2e3bdb089cea3","status":"passed","time":{"start":1743275115584,"stop":1743275119037,"duration":3453}}]},"3f4879a68edd124ba1941d13283ca312":{"statistic":{"failed":0,"broken":1,"skipped":0,"passed":0,"unknown":0,"total":1},"items":[{"uid":"9b48cf1d93468078","status":"broken","statusDetails":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=4723): Max retries exceeded with url: /session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x103a4e780>: Failed to establish a new connection: [Errno 61] Connection refused'))","time":{"start":1743275076385,"stop":1743275076385,"duration":0}}]},"23090410a9618e4070b272c264ba0311":{"statistic":{"failed":0,"broken":0,"skipped":0,"passed":1,"unknown":0,"total":1},"items":[{"uid":"ad69ef9a20b2641f","status":"passed","time":{"start":1743275087386,"stop":1743275095192,"duration":7806}}]},"60c51a4e5f869a081307c23c096196ef":{"statistic":{"failed":0,"broken":0,"skipped":0,"passed":1,"unknown":0,"total":1},"items":[{"uid":"41db2ef005c2f82c","status":"passed","time":{"start":1743275100832,"stop":1743275109984,"duration":9152}}]}}